README.md

```diff
@@ -2,36 +2,40 @@
-    <br /><br />
+    <a href="https://t.me/adguarden">Telegram</a>
+    <br>
 </p>
 
 <p align="center">
-<image src="https://cdn.adtidy.org/content/github/vpn/ios/vpn_disconnected.png" width="250">  <image src="https://cdn.adtidy.org/content/github/vpn/ios/vpn_locations.png" width="250">  <image src="https://cdn.adtidy.org/content/github/vpn/ios/vpn_connected.png" width="250">
+  <image src="https://cdn.adtidy.org/content/github/vpn/ios/vpn_disconnected.png" width="250">
+  <image src="https://cdn.adtidy.org/content/github/vpn/ios/vpn_locations.png" width="250">
+  <image src="https://cdn.adtidy.org/content/github/vpn/ios/vpn_connected.png" width="250">
 </p>
 
 > **Disclaimer**
-> * AdGuard VPN for Windows is not an open source project. We use GitHub as an open bug tracker for users to see what developers are working on. However, we at AdGuard create [a lot of open source software](https://github.com/search?o=desc&q=topic%3Aopen-source+org%3AAdguardTeam+fork%3Atrue&s=stars&type=Repositories).
+> * AdGuard VPN for iOS is not an open-source project. We use GitHub as an open bug tracker for users to see what developers are working on. However, we at AdGuard create [a lot of open-source software](https://github.com/search?o=desc&q=topic%3Aopen-source+org%3AAdguardTeam+fork%3Atrue&s=stars&type=Repositories).
 > * Privacy policy: https://adguard-vpn.com/privacy.html
 
-AdGuard VPN serves to protect your online privacy, encrypt your connection, hide your IP address and websites you visit from anyone (including your ISP). Cutting-edge technologies and multiple customization options will help conceal your location and ensure anonymous browsing on the web.
+AdGuard VPN serves to protect your online privacy, encrypt your connection, and hide your IP address and websites you visit from anyone (including your ISP). Cutting-edge technologies and multiple customization options will help conceal your location and ensure anonymous browsing on the web.
 
-To get more information and to download AdGuard VPN for iOS, visit our website [https://adguard-vpn.com/](https://adguard-vpn.com/).
+To get more information and to download AdGuard VPN for iOS, visit [our website](https://adguard-vpn.com/).
 
 <a id="feedback"></a>
 
 ## Feedback options
 
-We appreciate your feedback and always welcome both constructive critisism and new ideas.
+We appreciate your feedback and always welcome both constructive criticism and new ideas.
 
-You can use GitHub to report a bug or to submit a feature request. To do so, go to [this page](https://github.com/AdguardTeam/AdguardVPNForios/issues), click the _New issue_ button and choose between creating a bug report or feature request.
+You can use GitHub to report a bug or to submit a feature request. To do so, go to [this page](https://github.com/AdguardTeam/AdGuardVPNForiOS/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc), click the _New issue_ button and choose between creating a bug report or feature request.
```

Comment:
For this section, better to do the content like in Android projects.

Reply:
Done.